scripts/zap-scanner.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/sh
+
+
+TARGET=$1
+ENVIRONMENT=$2
+
+TARGET_DIR="$(pwd)/tdrs-$TARGET"
+REPORT_NAME=owasp_report.html
+REPORTS_DIR="$TARGET_DIR/reports"
+
+if [ "$ENVIRONMENT" = "nightly" ]; then
+    APP_URL="https://tdp-$TARGET-staging.app.cloud.gov/"
+    CONFIG_FILE="zap.conf"
+elif [ "$ENVIRONMENT" = "circle" ]; then
+    CONFIG_FILE="zap.conf"
+    if [ "$TARGET" = "frontend" ]; then
+        APP_URL="http://tdp-frontend/"
+    elif [ "$TARGET" = "backend" ]; then
+        APP_URL="http://web:8080/"
+    else
+        echo "Invalid target $TARGET"
+        exit 1
+    fi
+elif [ "$ENVIRONMENT" = "local" ]; then
+
+    # docker-compose down
+    # docker-compose up -d --build
+
+    if [ "$TARGET" = "frontend" ]; then
+        APP_URL="http://tdp-frontend/"
+    elif [ "$TARGET" = "backend" ]; then
+        APP_URL="http://web/"
+    else
+        echo "Invalid target $TARGET"
+        exit 1
+    fi
+else
+    echo "Invalid environment $ENVIRONMENT"
+    exit 1
+
+fi
+
+echo $TARGET
+echo $ENVIRONMENT
+
+echo $TARGET_DIR
+echo $REPORT_NAME
+echo $REPORTS_DIR
+
+echo $APP_URL
+echo $CONFIG_FILE
+
+if command -v sha256sum >/dev/null ; then
+	SHASUM=sha256sum
+elif command -v shasum >/dev/null ; then
+	SHASUM=shasum
+fi
```

Comment:
This block appears to be unused.

Reply:
Thats what I thought as well.

Reply:
@riatzukiza @jtwillis92 how do you want to handle this? create a ticket to clean it up? add it as a task to another related ticket or fix here?

Reply:
Looks like it was removed in a recent commit, which I agree is the correct move.